packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -16,7 +19,7 @@ def lb_enabled():
     return config['enable_lb'] == 'true'
 
 
-def ensure_routable(cmd, service_points, timeout=120):
+def ensure_routable(cmd, service_points, timeout=180):
```

Comment:
This function should be fixed up. It makes no sense to return a callable function in this usage context. Rather, get ride of _ensure_routable entirely and just put this decorator at the top level. Thus, you just need to call ensure_routable(cmd, service_points) to evaluate this and nothing will be lost